pkg/v3/types/basetypes.go

```diff
@@ -172,6 +172,11 @@ type UpkeepPayload struct {
 	CheckData []byte
 }
 
+// Determines whether the payload is empty, used within filtering
+func (p UpkeepPayload) IsEmpty() bool {
```

Comment:
Should we add a TODO here for comparing more fields in future if needed?

Reply:
I can add the fields right now but wasn't sure they were needed since wid should be the unique identifier for this struct. wdyt?

Reply:
discussed offline, keeping just workID for now